hack/make-rules/test-integration.sh

```diff
@@ -41,27 +41,33 @@ KUBE_TEST_ARGS=${KUBE_TEST_ARGS:-}
 kube::test::find_integration_test_dirs() {
   (
```

Comment:
Uh, I didn't change this.  Not sure I want to tackle cleaning this up more than I already have.